<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Shangchuan</class>
 <widget class="QWidget" name="Shangchuan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>741</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="returnbtn">
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>从</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="FromDataEdt"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>到</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="ToDataEdt"/>
    </item>
    <item>
     <widget class="QPushButton" name="searchBtn">
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>28</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="UpdateBtn">
      <property name="text">
       <string>上传</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="ResTabWid">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>740</width>
     <height>400</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>720</width>
     <height>400</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>750</width>
     <height>400</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <column>
    <property name="text">
     <string>日期</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>上传文件</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>上传结果</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>returnbtn</sender>
   <signal>clicked()</signal>
   <receiver>Shangchuan</receiver>
   <slot>Return()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchBtn</sender>
   <signal>clicked()</signal>
   <receiver>Shangchuan</receiver>
   <slot>Search()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>432</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>529</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UpdateBtn</sender>
   <signal>clicked()</signal>
   <receiver>Shangchuan</receiver>
   <slot>Upload()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>723</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>779</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Return()</slot>
  <slot>Search()</slot>
  <slot>Upload()</slot>
 </slots>
</ui>
